pmdarima/pipeline.py

```diff
@@ -442,28 +443,24 @@ def _post_predict(self, Xt, return_vals, return_conf_int,
                       inverse_transform):
         """Inverse-transform predictions to original data scale"""
 
-        # we don't currently support this... it will require an API change
-        # to also inverse-transform the 2-d confidence intervals
-        # TODO: fix this ^
-        if inverse_transform and return_conf_int:
-            warnings.warn("Inverse transformation on confidence intervals not "
-                          "currently supported, will not inverse transform",
-                          UserWarning)
-            inverse_transform = False
-
         if not inverse_transform:
             return return_vals
 
-        # TODO: support inverse transform on confidence intervals
         y_pred = return_vals
         conf_ints = None
         if return_conf_int:
             y_pred, conf_ints = y_pred
+            Xt_lower_bound, Xt_upper_bound = Xt, Xt
 
         # step through transformers in the reverse order
         for name, transformer in self.steps_[::-1]:
             if isinstance(transformer, BaseEndogTransformer):
                 y_pred, Xt = transformer.inverse_transform(y_pred, Xt)
+                if return_conf_int:
+                    conf_ints[:, 0], Xt_lower_bound = transformer.inverse_transform(
+                        conf_ints[:, 0], Xt_lower_bound)
+                    conf_ints[:, 1], Xt_upper_bound = transformer.inverse_transform(
+                        conf_ints[:, 1], Xt_upper_bound)
```

Comment:
This is a bit tricky, since the `Xt` result of the inverse transform will either be `Xt` (returned unchanged to match the signature in the case of an endog transformer) or `Xt_prime`, which is inverse-transformed irrespective of `y`.

In either case, you shouldn't need to worry about `Xt_lower_bound` and `Xt_upper_bound` here. I think you could achieve the same thing like:
```python
                    conf_ints[:, 0], Xt = transformer.inverse_transform(
                        conf_ints[:, 0], Xt)
                    conf_ints[:, 1], _ = transformer.inverse_transform(
                        conf_ints[:, 1], Xt)
```

Reply:
@tgsmith61591 is this a typo in your response? The current implementation, does not reassign `Xt` on the first transformation, but yours does. I don't understand this enough to know which is right 🤷‍♂️

Reply:
Xt is reassigned when computing y_pred (at 458). That's the first transformation in the loop.